invenio_banners/resources/config.py

```diff
@@ -19,6 +19,8 @@ class BannerServerSearchRequestArgsSchema(SearchRequestArgsSchema):
     """Banner request parameters."""
 
     sort_direction = ma.fields.Str()
+    active = ma.fields.Bool()
+    url_path = ma.fields.Str()
```

Comment:
This is just me being inquisitive 😜 .

I believe `request_extra_args` was there initially to do what you're trying to do, but it was probably left behind forgotten.
Since `request_extra_args` is not used anywhere (the main reason I landed in this PR), should we take the opportunity to remove it? WDYT?

Reply:
Hi Esteban! I think it is a very good idea, I cannot accommodate it now as a part of this PR unfortunately, I need it for my upcoming deployment of ILS (it is a part of a bigger chain of changes). Shall we create a ticket?